README.md

```diff
@@ -2,38 +2,77 @@
 
 The ArDrive Command Line Interface (CLI) Beta contains all of the needed public file synchronization and Arweave wallet capabilities via a Node.js application. This works in unison with the ArDrive Web App.
 
-This app is currently in beta testing, and as such may have bugs and does not offer Private/Encrypted Drives.
+To use the ArDrive CLI, install it with your favorite package manager, and run "ardrive-cli". As of now, the local SQLite database will be created in the directory that you run the CLI in.
 
-To use the ArDrive CLI, install it with your favorite package manager, and run "ardrive-cli". As of now, the local SQLite database will be created in the directory that you run the CLI in. If you are experiencing permissions issues (Mac OSX), you may also need to place your wallet file in the same directory you are running ardrive-cli
+If you are experiencing permissions issues (Mac OSX), you may also need to place your wallet file in the same directory you are running ardrive-cli
 
-## Setup
+## Developer Setup
+
+Follow these steps to get the developer environment up and running:
+
+### Install Yarn 2
+
+Both the ArDrive CLI and ArDrive Core JS use Yarn 2, so install the latest version with the [yarn installation instructions][yarn-install]. In most cases:
+
+```shell
+# Brew:
+brew install yarn
+
+# Or with NPM:
+npm install -g yarn
+```
+
+We also use husky. To enable hooks locally, you will need to run:
+
+```shell
+yarn husky install
+```
 
 ### NVM
 
-This repo uses NVM and an `.nvmrc` file to lock the Node version to the current version used by `ardrive-core-js`. Follow these steps to get NVM up and running on your system:
+This repository uses NVM and an `.nvmrc` file to lock the Node version to the current version used by `ardrive-core-js`.
+
+**Note for Windows: We recommend using WSL for setting up NVM on Windows using the [instructions described here][wsl-install]**
+
+Follow these steps to get NVM up and running on your system:
 
 1. Install NVM using [these installation instructions][nvm-install].
 2. Navigate to this project's root directory
 3. Ensure that the correct version of Node is installed by performing: `nvm install`
 4. Use the correct version of Node, by using: `nvm use`
 
-### We use Yarn 2
+### ArDrive-Core-JS
 
-```
-$ yarn install
-$ ardrive-cli
-```
+You'll need a build of the `ardrive-core-js` library on your local system to run the CLI:
```

Comment:
This could be added as advice to test a local version of Ardrive Core. e.g. if you want to validate a PR

Otherwise we should be importing a published ardrive-core package. That line on `package.json `was there to ease our dev flow but should never reach prod.